src/binding.cc

```diff
@@ -2029,6 +2034,38 @@ MaybeBool v8__Module__SetSyntheticModuleExport(const v8::Module& self,
       isolate, ptr_to_local(export_name), ptr_to_local(export_value)));
 }
 
+struct WasmStreamingSharedPtr {
+  std::shared_ptr<v8::WasmStreaming> inner;
+};
+
+static_assert(sizeof(WasmStreamingSharedPtr) <= 2 * sizeof(void*),
```

Comment:
Why <= ?

Reply:
`std::shared<T>` is normally two pointers big but it's conceivable that an alternative libc++ might implement it as a single pointer. IOW, over-allocating is okay, under-allocating is not.